config.properties

```diff
@@ -2,5 +2,5 @@
 # No quotation marks are required
 [DEFAULT]
 merchantAccount =
-apiKey =
-clientKey =
+apiKey = 
```

Comment:
Let us add config.properties to the .gitignore file and remove it from the commited files because if someone pushes by mistake it will push his API key and create a security breach. Maybe rename this file to config.properties.example

Reply:
I used "git update-index --skip-worktree <file>" for the config.properties so local changes to the file won't be commited. Let me know if you think that's a good solution.